source/com.microsoft.tfs.client.common/src/com/microsoft/tfs/client/common/git/utils/GitHelpers.java

```diff
@@ -30,7 +30,7 @@ public static boolean isEGitInstalled(final boolean activateEGit) {
         } else if (activateEGit) {
             return activateBundle(egit_core) && activateEGitUI();
         } else {
-            return false;
```

Comment:
It would be good here to have a comment that explains how we got into this else clause and why we should return true.